content/timeline.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+permalink: /timeline/
+title: Timeline 
+class: about anchors-center-fix
+---
+
+Since their initial creation in 2016, the CWL standards have gained much traction and are currently widely supported in practice. Below is a timeline of key moments in the evolution of the CWl Project and standards. 
```

Comment:
```suggestion
Since their initial creation in 2016, the CWL standards have gained much traction and are currently widely supported in practice. Below is a timeline of key moments in the evolution of the CWL project and standards. 
```